### ISSUE TITLE
Polygon Hackathon Bounty - Automations for Mai.Finance

### ISSUE THREAD

**Issue by kilaqidao** [kilaqidao/bounty#1]
##**"How might we use automation within Mai.finance to improve the user experience?"**

There are great strategies one can use with Mai.finance. That said there are a lot of moving parts users have to manually deal with to get maximum returns. Below are just a few examples of ways automation could help users. 

**Example 1:**  You can use Aave market tokens as collateral on mai.finance, however there are multiple steps you would have to take to implement this strategy. 
**Example 2:** Auto repayment of loans - many users will use the tokens they earn on other farms to pay off their loans, currently this is a manual process. 
**Example 3**: Auto rebalancing or auto managing of vaults. On Mai.finance users can have multiple vaults for the same collateral type. This can be great for certain strategies, but can leave some vaults risker than others.  
**Example 4**: Auto re-vesting Qi rewards. Qi stakers earn protocol revenue every week on Wednesday. the more Qi a user has staked the more revenue they earn. To continue to re-vest your rewards this process is manually done. 

For greater DeFi adoption, education and easy of use is imperative. We would love to see what you can come up with to help make strategies on mai.finance or between mai.finance and partners more automated. 

You can use any tool you wish to create these automations such as, Instadapp assembly, Gelato, etc.

**## What is the QiDao Protocol?**

TL;DR: Zero-interest crypto lending!

Use the value of your crypto without selling it. It's as simple as throwing your crypto in a vault, and borrowing up to 75% of its value at 0% interest. Your loan is in MAI stablecoin, which you can use however you like, lever, farm, pay bills, the uses are endless”

Lending flow: create a QiDao vault, deposit your crypto assets, and start borrowing stablecoins against your collateral’s value, repay whenever you like

**## What is MAI (miMatic)?**

MAI is a stablecoin backed by collateral. MAI can only be minted with collateral backing it. MAI is created when users deposit accepted tokens as collateral in vaults and in turn receive a loan against that collateral. MAI is soft pegged to the USD, meaning that 1 MAI is around 1 USD.

**## What is Qi?**

Qi (pronounced CHEE) is the governance token of the QiDao Protocol. It allows those who hold it to vote on changes to the QiDao Protocol as well as earn protocol revenue.

Please visit our discord for more inspiration and mentorship from our community. 
Discord:https://discord.gg/ytWh4VkQmE"

You can also checkout our official documentation at: https://docs.mai.finance/
and our unofficial docs that contain community strategies at: https://qidao-qimps.gitbook.io/mai-finance-tutorials/

Best automation wins

**Comment by helloDoraHacks** [comment]
The issue has been marked as a [HackerLink Bounty](https://hackerlink.io/bounty/29).

* Bonus: **500** MAI

* Deadline: 2021-10-22T13:50:55.000Z

Record on the blockchain: `0x2349d683a337044abcb8aca5778998ffb2682a864172f10338bc440cc14849d9`

[on Chain Explorer](https://polygonscan.com/tx/0x2349d683a337044abcb8aca5778998ffb2682a864172f10338bc440cc14849d9)

**Comment by Goldselleruk** [comment]
[https://twitter.com/DigiByteCoin/status/1433418967339524104?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1433418967339524104%7Ctwgr%5E%7Ctwcon%5Es1_c10&ref_url=https%3A%2F%2Fmedium.com%2Fmedia%2F7ee707c520170890e9e686f29ff8470a](https://twitter.com/DigiByteCoin/status/1433289949114081285?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1433289949114081285%7Ctwgr%5E%7Ctwcon%5Es1_c10&ref_url=https%3A%2F%2Fmedium.com%2Fmedia%2Fbab5cf280c9d5e5a1c792c068715c4b5) #

**Comment by helloDoraHacks** [comment]
Bounty Expired
